src/main/java/com/synopsys/integration/jenkins/detect/extensions/global/DetectGlobalConfig.java

```diff
@@ -209,13 +209,7 @@ public FormValidation doTestBlackDuckConnection(@QueryParameter("blackDuckUrl")
             if (connectionResult.isFailure()) {
                 int statusCode = connectionResult.getHttpStatusCode();
                 String validationMessage;
-                try {
-                    String statusPhrase = EnglishReasonPhraseCatalog.INSTANCE.getReason(statusCode, Locale.ENGLISH);
-                    validationMessage = String.format("ERROR: Connection attempt returned %s %s", statusCode, statusPhrase);
-                } catch (IllegalArgumentException ignored) {
-                    // EnglishReasonPhraseCatalog throws an IllegalArgumentException if the status code is outside of the 100-600 range --rotte AUG 2020
-                    validationMessage = "ERROR: Connection could not be established.";
-                }
+                validationMessage = getInstanceStatusPhrase(statusCode);
```

Comment:
determineValidationMessage is likely a better method name here, since we won't always be getting a status code phrase-- and it should probably be invoked on line 211 when we initialize validationMessage.